src/orion/algo/pbt/pbt.py

```diff
@@ -436,6 +456,7 @@ def _generate_offspring(self, trial):
             ]
 
             new_trial = trial_to_branch.branch(params=new_params)
+            # TODO: Keep this? or not?
             new_trial = self.space.transform(self.space.reverse(new_trial))
```

Comment:
I don't understand why I did not use self.format_trial. 

I think there is a problem here with the separate registries and the fact that the algorithm only see the trials in the transformed space. PBT copies over the working_dir of the trials based on their IDs, but the ID that the user will get is based on the params in the original space. So the user and PBT will work with different working_dirs.

I don't see how we could easily solve this if we get rit of transform(reverse()) here. :thinking: Maybe fixing this should be part of another PR.

Reply:
Resolved?